client/src/main.ts

```diff
@@ -1,15 +1,16 @@
 import Vue from 'vue'
 import App from './App.vue'
-import router from './router'
+import { getVueRouter } from '@/router'
 import store from './store'
 import { getConfig } from '@/plugins/getConfig'
-import vuetify from './plugins/vuetify'
+import Vuetify from "vuetify/lib"
```

Comment:
Please change " to '.

How much damage would be caused by removing the `'quotes': 'off'` rule in .eslintrc.js ?

Reply:
ah good catch, thanks! 

hmm i dunno, living on the edge 😆 
Whats the value in removing it?

Reply:
It would enforce consistent quote usage (as we have with the Entity UIs).

Reply:
oh sorry, i misunderstood. I thought you meant turn it to `off` 😁

Reply:
We can look at tightening the lint rules later.